docker-compose.yml

```diff
@@ -60,6 +60,12 @@ services:
     env_file:
       .env
 
+  data-integration:
+    build:
+      context: ./data-integration
+    volumes:
+      - data:/data
```

Comment:
do you also want the env here to have the database configuration ?

Reply:
I have now set the db parameters in the `.env` file, I don't know whether this is what you meant